apps/nextra/pages/en/build/move/book/SUMMARY.mdx

```diff
@@ -0,0 +1,46 @@
+# The Move Programming Language
```

Comment:
I think the files in the book directory are missing frontmatter for the title of the page. e.g.,

```
---
title: "Summary"
---
```

Reply:
The frontmatter isn't required it seems to fill them out automatically

Reply:
Yeah I think it uses the sidebar titles as a fallback if the frontmatter title isn't filled in